src/bot/parse/parseVariables.ts

```diff
@@ -0,0 +1,30 @@
+import yargs from "yargs"
+
+import { LoggerContext } from "src/logger"
+import { arrayify } from "src/utils"
+
+export async function parseVariables(ctx: LoggerContext, str: string): Promise<Record<string, string> | Error> {
```

Comment:
just moved out the parser of -v variables from https://github.com/paritytech/command-bot/blob/79223d9d98ae9c5be92f2affdb985deeb4666c8a/src/bot/parsePullRequestBotCommandLine.ts, the logic is the same